app/forms/response_form.rb

```diff
@@ -26,7 +26,7 @@ def validate_questions
     survey_questions_ids = survey.questions.map(&:id)
     questions_ids = questions.pluck(:id)
 
-    return if (survey_questions_ids & questions_ids) == questions_ids
+    return if (survey_questions_ids & questions_ids).to_set == questions_ids.to_set
```

Comment:
Regarding [this benchmark](https://stackoverflow.com/a/56702586/6339862), should we change to use the `sort!` instead?
```suggestion
    return if (survey_questions_ids & questions_ids).sort! == questions_ids.sort!
```

Reply:
After checking the logic again, I think it's better to update to
`(questions_ids & survey_questions_ids) == questions_ids`

Fixed in https://github.com/nimblehq/nimble-survey-web/pull/82/commits/1985c8b6ffb8e2e2271c59b9ec9622831e945e46